go/vt/vttablet/tabletserver/state_manager.go

```diff
@@ -473,6 +481,7 @@ func (sm *stateManager) unserveCommon() {
 	sm.te.Close()
 	sm.qe.StopServing()
 	sm.tracker.Close()
+	sm.throttler.Close()
```

Comment:
Should this not be the first call? Lines 95-96 tell us
```
	// Open must be done in forward order.
	// Close must be done in reverse order.
```

Reply:
fixed